### DISCUSSION TITLE
Missing docs for "Account" section in Wiki

### DISCUSSION THREAD

**Discussion by lukynmatuska** [rustdesk/rustdesk#5610] [Feature Request] [answered]
Please add article to wiki about section "Account" used at Desktop app.

**Comment by dinger1986** [comment]
Do you mean for pro?

**Comment by lukynmatuska** [comment]
Maybe, I'm not sure for what is this section used. So there should be some docs.

**Reply by dinger1986** [reply, marked as answer]
added as part of this PR https://github.com/rustdesk/doc.rustdesk.com/pull/278
